Tools/autotest/ArduPlane_Tests/CheckedDisarm/flaps.txt

```diff
@@ -0,0 +1,13 @@
+QGC WPL 110
```

Comment:
is this mission used?

Reply:
Yeah, for fly_home_land_and_disarm I think,.